src/components/VolumeChart/hooks/useProjectTimeline.ts

```diff
@@ -74,22 +77,19 @@ export function useProjectTimeline({
       ...blocks,
     },
   })
-  // TODO: const { data: v4QueryResult } = useSubgraphQuery(ProjectTlDocument, {
-  //   where: {
-  //     projectId,
-  //   },
-  // })
+  const { data: v4QueryResult } = useSubgraphQuery(ProjectTlDocument, {
```

Comment:
Both `useProjectTlQuery` and `useSubgraphQuery` are going to run though, when we only want one or the other to run.

Maybe the easiest fix is to set the `enabled` flag on each query, to only be enabled when the project has the correct version.

https://tanstack.com/query/v4/docs/framework/react/guides/disabling-queries

Reply:
Good catch, I've modified the definition of the `useSubgraphQuery` function so we can pass it an object with `enabled` like here:

<img width="498" alt="Screenshot 2024-08-08 at 10 42 30 am" src="https://github.com/user-attachments/assets/1352e21d-c93b-4458-af44-5f03d3d3e3e9">

And then have used the equivalent of `enabled` for Apollo on the old v1v2v3 query above (`skip`)